tests/test_bot.py

```diff
@@ -1433,14 +1433,10 @@ def test_camel_case_aliases(self, bot_class, bot_method_name, bot_method):
         assert camel_case_function is bot_method, f"{camel_case_name} is not {bot_method}"
 
     @bot_methods()
-    def test_coroutine_functions(self, bot_class, bot_method_name, bot_method):
+    def test_coroutine_functions(self, bot_class, bot_method_name, bot_method, monkeypatch):
```

Comment:
Is monkeypatch needed?

Reply:
must've added that accidentally. Was removed in later commits